eng/helix/helix.proj

```diff
@@ -22,7 +23,10 @@
     <IsExternal>true</IsExternal>
     <MaxRetryCount Condition="'$(MaxRetryCount)' == ''">2</MaxRetryCount>
     <HelixAccessToken Condition="'$(_UseHelixOpenQueues)' != 'true'">$(HelixApiAccessToken)</HelixAccessToken>
-
+    <IncludeDotNetCli>true</IncludeDotNetCli>
+    <DotNetCliPackageType>sdk</DotNetCliPackageType>
+    <DotNetCliVersion>$(NETCoreSdkVersion)</DotNetCliVersion>
```

Comment:
@alexperovich is looking for ways to get this working without importing the .NET SDK's Sdk.props file. What we're doing here is apparently unsupported and suppressing the warning will likely lead to other issues.

Reply:
Suggest waiting for @alexperovich to get dotnet/arcade#5748 in. At that point, you can remove this line and either wait for the next Arcade promotion to `.NET Eng - Latest` or update the Arcade dependencies from `.NET Eng - Validation` (short-cutting a validation ring). I slightly prefer the latter because Arcade validation of our repo doesn't run tests. But, I'm not sure we should merge this PR without also doing Arcade validation.

Reply:
BTW with @alexperovich's change, you also won't need the added `<Import>` [above](https://github.com/dotnet/aspnetcore/pull/23585/files#diff-500537596960313988af742d444b490dR2) nor the `$(DotNetCliPackageType)` setting.

Reply:
Cool that's fine, we need to still track down why IIS functionals are no longer working with this change, so sounds like this will mostly just work once that PR is in

Reply:
I recommend removing these two lines as well as the `<Import>`. The Helix SDK will default to using the SDK from the build when it submits the job, no reason to duplicate what it's doing.

Reply:
We won't see those changes immediately though right?  We need to get our side working irrespective of the helix sdk improvements.

Reply:
@HaoK  yes, not until it is merged and flows out.

Reply:
Yup, that's what I thought, so I'll keep the temporary changes here for now since that lets us prep the other changes, and I'll back them out and rebase once the arcade changes are in

Reply:
Now that @alexperovich got dotnet/arcade#5748 in, it won't be much longer before we can get that fix. Importantly his design changed slightly and you'll need the `<DotNetCliPackageType>sdk</DotNetCliPackageType>` line. Will still be able to remove the `<Import>` line and `$(DotNetCliVersion)` setting.